### ISSUE TITLE
bitbucket webhook payload on env variable

### ISSUE THREAD

**Issue by wawashra** [jenkinsci/bitbucket-branch-source-plugin#691]
### What feature do you want to see added?

Hi, Is any chance to have the bitbucket webhook payload inside env var ?
the are many cases we need to apply base on some information from the payload like send an email to all pr reviews user or triggered extra stage when pr has approved event key 

Thanks

**Comment by wawashra** [comment]
I saw the forked repo, but the question how I can have a ready build from it to install on my jenkins ? and how I can use the payload data inside my pipeline  ? Could you please recap the steps for that ^_^ 

Thanks

**Comment by nfalco79** [comment]
I do not think this is a good chose to expose detail of the repository for which end user could not have access.
Anyway if you want the payload install and setup [bitbucket-pullrequest-builder-plugin](https://github.com/jenkinsci/bitbucket-plugin) that does exactly what you are asking
